src/App.jsx

```diff
@@ -1,12 +1,24 @@
-import AddTask from "./components/AddTask"
+import { useState } from "react"
+import Footer from "./components/Footer"
+import TodoFilters from "./components/TodoFilters"
+import TodoInput from "./components/TodoInput"
+import TodoList from "./components/TodoList"
 
 function App() {
+  const [tasks, setTasks] = useState([])
+  const [filter, setFilter] = useState("All")
+
 
   return (
-    <>
-      <h1>My Todo</h1>
-      <AddTask />
-    </>
+    <div className="flex flex-col h-full bg-gray-50 px-18 py-10">
+      <div className="flex flex-col gap-14">
+        <h1 className="flex justify-center text-4xl text-blue-600">My Todo</h1>
+        <TodoInput tasks={tasks} setTasks={setTasks} />
+      </div>
+      <TodoFilters tasks={tasks} setTasks={setTasks} filter={filter} setFilter={setFilter} />
```

Comment:
The `tasks` and `setTasks` props are not used, however, they cause the component to re-render even when it's not needed

```suggestion
      <TodoFilters filter={filter} setFilter={setFilter} />
```